src/base.mli

```diff
@@ -101,3 +101,23 @@ val available : unit -> bool
 [@@@js.implem
 let available () =
   Js_of_ocaml.Js.Optdef.test Js_of_ocaml.Js.Unsafe.global##.mixpanel]
+
+val set_group :
+  group_key:string ->
+  group_ids:string list ->
+  ?callback:(Ojs.t -> unit) ->
+  unit ->
+  unit
+  [@@js.global "mixpanel.set_group"]
+
+module Group : sig
+  type group_id
+
+  (*Pas sûr de si c'est bien des properties*)
+  val get_group : group_key:string -> group_id:Ojs.t -> group_id
+    [@@js.global "mixpanel.get_group"]
+
+  val set :
+    group_id -> prop:Properties.t -> ?callback:(Ojs.t -> unit) -> unit -> unit
```

Comment:
```suggestion
    group_id -> prop:Properties.t -> ?callback:(Ojs.t -> unit) -> unit
```

Is the `unit` parameter necessary?

Reply:
Yes because the last given argument (callback) is optional so if we don't give a necessary argument as last argument before the return type the function construction will failed in the build.

Reply:
Hmm… You are right. Sorry for the irrelevant remark.